test/options_test.cc

```diff
@@ -107,6 +109,34 @@ TEST_F(OptionsImplTest, BadH2FlagThrows) {
       MalformedArgvException, "Couldn't find match for argument");
 }
 
+TEST_F(OptionsImplTest, H2Flag) {
```

Comment:
Please add `//` single line human comments above each `TEST_F`.

Reply:
Done in https://github.com/envoyproxy/nighthawk/pull/9/commits/b388e0fe4af9121aaa40c2c167fbd5d338bb4f41; also noticed that some duplication crept in unnoticed, probably because of me missing that while resolving merge conflicts.